### PR TITLE
Etl scripts

### PR DESCRIPTION
**Pull request by soopertramp**
Check the code

### CODE REVIEW

File: src/tools/etl_script_final.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#import necessary libraries 
+import pandas as pd
+import matplotlib.pyplot as plt
+
+#reading the data
+df = pd.read_csv('data/supermarket_sales.csv')
+df.head()
```
Comment (mitulds) [inline src/tools/etl_script_final.py]: you can remove this line `df.head()`

File: src/tools/etl_script_final.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# convert the 'date' column to datetime format
+df['date'] = pd.to_datetime(df['date'])
+
+# convert the 'time' column to datetime format
+#df['time'] = pd.to_datetime(df['time'])
```
Comment (mitulds) [inline src/tools/etl_script_final.py]: can remove these too, you can clean up the script

File: src/tools/etl_script_final.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+#splitting tables
+table1 = df[['invoice_id', 'branch', 'city', 'customer_type','gender','product_line', 'unit_price']]
+
+table2 = df[['invoice_id', 'quantity', 'tax_5_percent', 'total', 'date', 'time', 'payment']]
+
+table3 = df[['invoice_id', 'cogs', 'gross_margin_percentage', 'gross_income', 'rating',]]
+
+# Save the tables as CSV files
+df.to_csv('supermarket_sales_cleaned.csv', index=False)
```
Comment (mitulds) [inline src/tools/etl_script_final.py]: why do you need this one? its the original data

Comment (soopertramp) [reply src/tools/etl_script_final.py]: Original data has date column issues, so saved a cleaned ordered one

**Review by mitulds — CHANGES_REQUESTED** [review]
good to merge once changes are made.

**Pull request merged**